uint8 groupMask
bool relative
geometry_msgs/Point goal
float32 yaw # deg
duration duration
---
